index.js

```diff
@@ -5,6 +5,26 @@ const canvas = document.getElementById("stonkscanvas");
 const context = canvas.getContext("2d");
 const imageObj = new Image();
 const vowels = "aeiouy";
+const consonants = "";
+// rules have format [regex_to_replace, text_to_insert, priority]
+// rules are executed in a sorted manner by priortiy, rule regex length
+// default
+let rules = [
+  ["([a-z]+)(ch)", "$1hc", 1],
+  ["([aiouy])(c)", "$1n", 1],
+  ["nc", "nk", 1],
+  ["^c", "k", 1]
+];
+
+const stonksifyRules = rules.sort(function compare(a, b) {
+  if (a[2] > b[2]) return -1;
+  else if (a[2] < b[2]) return 1;
+  else {
+    if (a[0].length > b[0].length) return -1;
+    else if (a[0].length < b[0].length) return 1;
+    else return 0;
+  }
+});
```

Comment:
Can we add a comment what this does at a high level?

Reply:
sort have done this on 9 10 11, but will add more detail